templates/zerver/emails/followup_day1.source.html

```diff
@@ -53,7 +53,6 @@
 </p>
 
 <p>
-    {% trans support_email=macros.email_tag(support_email) %}Questions? Contact
-us any time at {{ support_email }}.{% endtrans %}
+    Questions? <a href="{{ support_email }}" >Contact us</a> — we'd love to help!
```

Comment:
Let's continue using the `{% trans %}` tags. Checkout the macro used used in the in the tag, you don't have to use `<a href="{{ support_email }}" >Contact us</a>`. 

> In [emails: Improve followup_day1 (registration confirmation) email. #24279](https://github.com/zulip/zulip/pull/24279), we are changing the followup_day1 contact us line to: "Questions? Contact us any time at {{ support_email }}." It should further be updated to:
> 
> * **Cloud**: Questions? [Contact us](https://zulip.com/help/contact-support) — we'd love to help!
> * **Self-hosted**: (same as self-hosted line above)

This part should be further updated for Cloud vs Self-hosted

Reply:
sure I will do it